libs/common/src/main/kotlin/logging/LogUtils.kt

```diff
@@ -3,7 +3,7 @@ package no.nav.etterlatte.libs.common.logging
 import org.slf4j.MDC
 import java.util.*
 
-const val X_CORRELATION_ID: String = "X-Correlation-Id"
+const val X_CORRELATION_ID: String = "X-Correlation-ID"
 const val CORRELATION_ID: String = "correlation_id"
```

Comment:
Hvorfor har vi begge to?

Reply:
nja - den vanlige konvensjonen er vel å ha en X-Correlation-ID som brukes som header i kall mellom apper, mens appenene internt har sin egen korrelasjonsid mellom logginnslag. Som du ser i `HttpHeaders` så er `X-Correlation-ID` og tilsvarende en uoffisiell standard-header. Når det er sagt så finnes det også noe som heter `Nav-Call-Id`. Mulig den også kan vurderes for dette?

Reply:
Jepp vi kan ha en internt for appen og en mellom apper, men `Nav-Call-Id` er den riktige korrelasjonside headeren ja.